LandSandBoat/server
# Add mp/cast/recast values for Tier 6 BLM JP spells and PLD/DRK JP Tier 2…

… spells

* Added mpCost/castTime/recastTime values for all 6 Tier 6 BLM Elemental spells (Stone VI, Water VI, etc..)
* Added mpCost/castTime/recastTime values for Tier 2 PLD spell Enlight II
* Added mpCost/castTime/recastTime values for Tier 2 DRK spell Endark II

All values as per BG-Wiki



**_I affirm:_**

## What does this pull request do?
Adds MP cost, cast times, and recast times for all 6 Tier 6 BLM Elemental spells (Stone VI, Water VI, Aero VI, Fire VI, Blizzard VI, Thunder VI) as well as the Tier 2 PLD spell Enlight II as well as the Tier 2 DRK spell Endark II

Closes #1980 

## Steps to test these changes
Load up a personal server **before** this PR, use !addallspells, cast any of the above spells and notice that the tier 6 BLM spells/Tier 2 PLD/DRK spells mentioned cost zero MP and have no cast time/recast time.

Load up a personal server **after** this PR, use !addallspells, cast any of the above spells and notice that the same spells now cost the correct amounts of MP and are also taking the correct amount of time to initial cast/recast.
